proto/info.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto2";
+option optimize_for = LITE_RUNTIME;
+package valhalla;
+
+message Statistic {
+  optional string name = 1;  // the name of the statistic
+  optional double value = 2; // the value of the statistic
+}
+
+message Info{
```

Comment:
this should be a place where we can add statistical information about the processing of an api call or request. for now its just key value pairs

Reply:
This could also be a place to add metadata about the type of statistic contained within. Eg if the stat is a timer, a count etc.